io-engine-tests/src/nexus.rs

```diff
@@ -504,6 +504,50 @@ impl NexusBuilder {
         }
     }
 
+    pub async fn wait_children_online2(
```

Comment:
Why not add the extra string printing logic into existing `wait_children_online` function itself?

Reply:
Sorry, I forgot to clean up tmp debug stuff.